lib/nexpose/password_policy.rb

```diff
@@ -0,0 +1,58 @@
+module Nexpose
+  require 'json'
+  # Configuration structure for password policies.
+  class PasswordPolicy < APIObject
+
+    attr_accessor :policyName
+    attr_accessor :minLength
+    attr_accessor :maxLength
+    attr_accessor :capitals
+    attr_accessor :digits
+    attr_accessor :specialChars
+
+    def initialize(policyName:, minLength:, maxLength:, specialChars:, capitals:, digits:)
+      @policyName = policyName.to_s
+      @minLength = minLength.to_i
+      @maxLength = maxLength.to_i
+      @specialChars = specialChars.to_i
+      @capitals = capitals.to_i
+      @digits = digits.to_i
+    end
+
+    def self.from_hash(hash)
+      new(hash[:policyName],
```

Comment:
I just realized this probably doesn't work with the keyword arguments we moved to. We'd need to update this new call to use the keys. Better yet you can use a kwargs (keyword args) splat.

```ruby
new(policyName: hash[:policyName],
    # etc...
    capitals: hash[:captials])
```